2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -659,7 +659,7 @@
  {
   "input": "Also, for those of you who enjoy exercises, I'll leave one up on the screen for how you would handle the case of two different standard deviations.",
   "model": "nmt",
-  "translatedText": "Además, para aquellos de ustedes que disfrutan de los ejercicios, dejaré uno en la pantalla sobre cómo manejarían el caso de dos desviaciones estándar diferentes.",
+  "translatedText": "Además, para los que les gustan los ejercicios, pondré uno en la pantalla sobre cómo se puede hacer en el caso de dos desviaciones estándar distintas.",
```

```suggestion
  "translatedText": "Además, para los que os gustan los ejercicios, pondré uno en la pantalla sobre cómo se puede hacer en el caso de dos desviaciones estándar distintas.",
``` 
Shouldn't we keep it in the 2nd person here?